server/src/controllers/rsvpController.ts

```diff
@@ -33,7 +46,26 @@ export const deleteRsvp = async (req: Request, res: Response, next: Function) =>
   }
 };
 
+export const updateRsvpByTokenController = async (req: Request, res: Response, next: Function) => {
+  try {
+    const { id: eventId, rsvpId } = req.params;
+    const { token, name, status, guests } = req.body;
+    console.log(req.body, req.params)
+    if (!token || !name || !status === undefined) {
```

Comment:
```suggestion
    if (!token || !name || !status) {
```

`undefined` unnecessary?

Reply:
ah yeah, also need to remove the log statement 😅